app/javascript/packs/confirmationEmail.js

```diff
@@ -0,0 +1,30 @@
+function showConfirmationEmailModal() {
+  window.Forem.showModal({
+    title: "Didn't get the email?",
+    contentSelector: '#confirm-email-modal',
+    overlay: true,
+    onOpen: attachHandlers,
+  });
+}
+
+function attachHandlers() {
+  const dismissModalButtons =
+    document.getElementsByClassName('js-dismiss-button');
+
+  if (dismissModalButtons.length > 1) {
```

Comment:
With the nature of how the modals are implemented, it duplicates the content of `#confirm-email-modal` and adds it to the window within `#window-modal`. This means that there are two of the same elements in the DOM, hence we use the last one that is added to the DOM.

Reply:
Can you transform this into a comment inside the code? 😊

Reply:
Sounds good, I'll add this!